templates/sshd.ubuntu18.erb

```diff
@@ -0,0 +1,15 @@
+@include common-auth
```

Comment:
@KlavsKlavsen, can you add update the template to reflect the [addition](https://github.com/ghoneycutt/puppet-module-pam/pull/160/commits/32d27a53c2824b7c2b619950361815917a84bece#diff-e3cdb1800313bb4fcca541886851296bR3) that was done by @fbarbeira for sshd.ubuntu18.erb?